simglucose/analysis/report.py

```diff
@@ -92,17 +92,21 @@ def percent_stats(BG, ax=None):
     return p_stats, fig, ax
 
 
-def risk_index_trace(df_BG, visualize=False):
-    chunk_BG = [df_BG.iloc[i:i + 60, :] for i in range(0, len(df_BG), 60)]
+def risk_index_trace(df_BG, sample_rate, visualize=False):
```

Comment:
Thank you for this parameterization, but sample_rate here is a bit misleading. I believe it should be step_size or something similar. How about `def risk_index_trace(df_BG, window_length: int = 60, sample_time: int  = 3, visualize=False)`, where window_length is in minutes , so step_size = int(window_length / sample_time). Then you can do chunk_BG = [df_BG.iloc[i:i + step_size, :] for i in range(0, len(df_BG), step_size)].

Reply:
Sure, agree that window length can also be parameterized:D